### ISSUE TITLE
Upgrade espanso

### ISSUE THREAD

**Issue by gtback** [gtback/dotfiles#4]
https://espanso.org/docs/migration/overview/

**Comment by gtback** [comment]
Ran:

```
espanso stop
espanso unregister
brew remove espanso
```

**Comment by gtback** [comment]
```
brew untap federico-terzi/espanso
```

**Comment by gtback** [comment]
```
brew install --cask espanso
```

**Comment by gtback** [comment]
Fixed by dae6e3e687f254810c59151cc86688cc886acad7
